common/pkg/capi/clustertopology/handlers/lifecycle/interfaces.go

```diff
@@ -9,6 +9,11 @@ import (
 	runtimehooksv1 "sigs.k8s.io/cluster-api/exp/runtime/hooks/api/v1alpha1"
 )
 
+const (
+	// LifecycleRetryAfterSeconds is used for BeforeClusterUpgrade hooks in case of failure.
+	LifecycleRetryAfterSeconds = 30
```

Comment:
Since the package is `lifecycle`, we  can simplify the name to
```suggestion
	RetryAfterSeconds = 30
```

However, _if this is value is only for BeforeClusterUpgrade_, then please consider renaming it to
```suggestion
	BeforeClusterUpgradeRetryAfterSeconds = 30
```